CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This file documents all notable changes to the GEOS-Chem repository starting in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - TBD
+### Added
+- Added createRunDir for processed cubed-sphere GEOS-IT meteorology
```

Comment:
Change to "Added run directory creation for processed cubed-sphere GEOS-IT meteorology". As is, it reads as if a new createRunDir was added, when the existing one was just modified.

Reply:
Done